swagger.yaml

```diff
@@ -751,7 +751,99 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
-  "/vacationsRequests":
+  "/severa/projects/{severaProjectGuid}/phaseswithhierarchy":
```

Comment:
I'm not sure about the endpoint name of phaseswithhierarchy, I'd suggest just phases

Reply:
There is no endpoint "phases" for user. this endpoint receives all phases for a particular user. Do you want us to retrieve all phases and then sort the ones that we need?

Reply:
All phases of a particular project* 
Not user